src/ground_script_1.c

```diff
@@ -0,0 +1,1863 @@
+u32 ShiftItemsDownFrom();
+u32 sub_80920B8();
+void sub_809733C(s16, bool8);
+void sub_80973A8(s16, bool8);
+void sub_8097418(s16, bool8);
+void sub_80975A8(s16, bool8);
+void sub_80976F8(u8);
+u32 sub_8098100();
+void sub_8098D1C(s32, u32, s32);
+void sub_8098D80(s32);
+void sub_8098E18(s32, s32);
+u32 sub_80999E8();
+u32 sub_80999FC();
+u32 sub_8099A10();
+u32 sub_8099A34();
+u32 sub_8099A48();
+u32 sub_8099A5C();
+u32 sub_8099AFC();
+u32 sub_809A6E4();
+u32 sub_809A6F8();
+u32 sub_809A738();
+u32 sub_809A768();
+bool8 ScriptPrintNullTextbox(void);
+bool8 ScriptPrintEmptyTextbox(void);
+u32 sub_809A83C();
+u32 sub_809AC7C();
+u32 sub_809ADD8();
+bool8 ScriptPrintText(s32, s16, char*);
+bool8 sub_809AEEC(char*);
+bool8 sub_809AF2C(char*);
+bool8 sub_809AF6C(s16, char*);
+void sub_809AFC8(bool8, s32, s32, char*);
+u32 sub_809B028();
+bool8 sub_809B1C0(s32, s32, char[12]);
+void sub_809B1D4(u8, s32, s32, char*);
+void sub_809C770(s16, s16);
+s32 sub_809CBA4(u8);
+u32 sub_809CC90();
+void sub_809D0BC(void);
+void sub_809D124(s32, s32, s32);
+void sub_809D158(s32, Position32*);
+void sub_809D170(s32, s32);
+void sub_809D190(s32, Position32*, s32);
+void sub_809D1A8(s32, s32, s32);
+void sub_809D1CC(s32, Position32*, s32);
+void sub_809D1E4(s32, s32, s32);
+void sub_809D208(s32, Position32*, s32);
+void sub_809D220(s32, s32, s32);
+u32 sub_809D52C();
+u32 InitScriptData();
+void sub_809D710(Action*, ScriptInfoSmall*, s16);
+u8 sub_809D8EC(Action*, s16);
+bool8 sub_809D940(void);
+void sub_809D9B8(s16);
+bool8 sub_809DA08(Action*, s16, u8);
+ScriptCommand *sub_80A242C(Action*, s32);
+ScriptCommand *sub_80A2460(Action*, s32);
+s16 sub_80A4D7C(s32);
+s16 sub_80A7AE8(s16);
+void sub_80A87AC(s32, s32);
+void sub_80A8BD8(s16, s32*);
+u32 sub_80A8C2C();
+u32 sub_80A8C98();
+char *sub_80A8D54(s16);
+s16 sub_80A8FD8(s32, Position32*);
+s16 sub_80A8F9C(s32, Position32*);
+u32 sub_80A9050();
+u32 sub_80A9090();
+s16 sub_80AC448(s16, Position32*);
+s32 sub_80AC49C(s16, Position32*);
+s16 sub_80AD360(s16, Position32*);
+s16 sub_80AD3B4(s16, Position32*);
+u32 sub_80AD8B4();
+u32 sub_80AD914();
+void DeleteBlankGroundEvents(void);
+void DeleteBlankGroundLives(void);
+void DeleteBlankGroundObjects(void);
+void DeleteBlankGroundEffects(void);
+
+extern int gFormatData_202DE30[10];
+
+extern s16 gCurrentMap;
+extern s16 gUnknown_2039A32;
+extern s16 gUnknown_2039A34;
+
+extern struct { char *unk0; s32 unk4; } gUnknown_2039D50[9];
+extern char gUnknown_2039D98[12];
+extern int gUnknown_203B4AC;
+
+extern FunctionScript gFunctionScriptTable[];
+
+extern Position32 gUnknown_81164DC;
+extern char gUnknown_81165D4[];
+extern char gUnknown_81165F4[];
+extern char gUnknown_811660C[];
+extern char gUnknown_8116628[];
+extern char gUnknown_8116644[];
+extern char gUnknown_8116664[];
+extern char gUnknown_8116684[];
+extern char gUnknown_81166C0[];
+extern char gUnknown_81166D8[];
+
+
+extern DebugLocation gUnknown_81166B4;
+
+/* No decompiler yet implemented for arm32 */
```

Comment:
Do we need this comment? It looks like just the comment decomp.me adds into new scratches.